site/jest.config.ts

```diff
@@ -27,8 +28,18 @@ module.exports = {
         ],
       },
       testEnvironment: "jsdom",
+      testEnvironmentOptions: {
+        customExportConditions: [""],
+      },
       testRegex: "(/__tests__/.*|(\\.|/)(test|spec))\\.tsx?$",
-      testPathIgnorePatterns: ["/node_modules/", "/e2e/"],
+      testPathIgnorePatterns: [
+        "/node_modules/",
+        "/e2e/",
+        // TODO: This test is timing out after upgrade a few Jest dependencies
+        // and I was not able to figure out why. When running it specifically, I
+        // can see many act warnings that may can help us to find the issue.
+        "/usePaginatedQuery.test.ts",
```

Comment:
@Parkreiner I may need your help with this 🙏 I created a separated issue to not block this one https://github.com/coder/coder/issues/12598

Reply:
No worries! Happy to take a look tomorrow after Backstage launch

Reply:
Oh, I didn't realize that was an option. Definitely think we should go with that

Reply:
fyi, https://jestjs.io/docs/api#testskipfailingname-fn-timeout :)

Reply:
For some reason, `describe.skip` or `test.failure` are not working 😢 it just hangs forever 

https://github.com/coder/coder/assets/3165839/ab8d2411-4773-4d99-bff2-2e00b5e61d02

Ps: I tried to post a longer video but GH didn't allow me because of the size.

Reply:
that's annoying... what happens if you add an empty non-skipped test?

Reply:
it just hangs... I think there is some weird thing happening on this test 🤔